### PR TITLE
Prepare CHANGELOG for release 1.5.1

### PR DESCRIPTION
**Pull request by dbrgn**
Here's the preliminary changelog for release 1.5.1. Did I forget anything?

As mentioned in #276, I plan to release 1.5.1 by the end of the week.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,41 @@ Possible log types:
...
+- [fixed] Do not page output of `tldr--update` ([#231])
+- [fixed] Handle cache download errors ([#253])
+- [fixed] Create macOS release builds with bundled root certificates ([#272])
+- [fixed] Clean up and fix shell completions ([#262])
+- [chore] Update dependencies ([#271], [#287])
+- [chore] Use anyhow for error handling ([#249])
+- [chore] Move shell completion scripts to their own directory ([#259])
+- [chore] Switch to Rust 2021 edition ([#284])
+
+Contributors to this version:
```
Comment (niklasmohrin) [inline CHANGELOG.md:36]: Let's also add Hans Gaiser from the initial cache config PR

**Review by niklasmohrin — COMMENTED** [review]
Did we not deprecate anything this year? 🤔 I am in favor of calling it 1.6 over 1.5.1, there is the completion stuff, we bumped msrv, there are new features, etc.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,41 @@ Possible log types:
...
+- [added] Add support for android platform ([#274])
+- [added] Add custom pages to list output ([#285])
+- [fixed] Cache: Return error if HTTP client cannot be created ([#247])
+- [fixed] Do not page output of `tldr--update` ([#231])
+- [fixed] Handle cache download errors ([#253])
+- [fixed] Create macOS release builds with bundled root certificates ([#272])
+- [fixed] Clean up and fix shell completions ([#262])
+- [chore] Update dependencies ([#271], [#287])
+- [chore] Use anyhow for error handling ([#249])
+- [chore] Move shell completion scripts to their own directory ([#259])
```
[the remark this answers is not in the document]
Comment (dbrgn) [reply CHANGELOG.md:33]: Fair enough!

File: CHANGELOG.md [hunk 3]
```diff
@@ -326,4 +368,18 @@ Thanks!
 [i215]: https://github.com/dbrgn/tealdeer/pull/215
 [i217]: https://github.com/dbrgn/tealdeer/pull/217
 [i227]: https://github.com/dbrgn/tealdeer/pull/227
+[#231]: https://github.com/dbrgn/tealdeer/pull/231
```
[the remark this answers is not in the document]
Comment (dbrgn) [reply CHANGELOG.md:371]: Yeah, if the value and the label are identical, we can leave away the label.

So this:

    - [added] Support custom pages and patches ([#142][i142])

...becomes

    - [added] Support custom pages and patches ([#142])

...which is simpler 🙂 For some reason I didn't think of this when I initially started the changelog.

I didn't bother to update previous labels though.

**Comment by dbrgn** [comment]
Closing this, I'll create a separate release PR.

**Pull request closed without merging**